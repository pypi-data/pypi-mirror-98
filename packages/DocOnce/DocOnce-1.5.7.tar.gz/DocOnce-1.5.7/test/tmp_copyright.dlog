running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'pdflatex', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to pdflatex
found info about 1 exercises
output in tmp_copyright.tex
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'pdflatex', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|CC BY}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to pdflatex
found info about 1 exercises
output in tmp_copyright.tex
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'pdflatex', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|CC BY-NC}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to pdflatex
found info about 1 exercises
output in tmp_copyright.tex
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'pdflatex', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|Released under the MIT license.}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to pdflatex
found info about 1 exercises
output in tmp_copyright.tex
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'sphinx', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to sphinx
*** error: heading "===== Exercise: My problem =====" is subsection and
    comes after chapter, but will be treated as section in sphinx
found info about 1 exercises
*** warning: new !split inserted (override all existing !split)
    before every ========= heading =========
    because this strategy gives a well-functioning
    table of contents in Sphinx
    (use --sphinx_keep_splits to enforce your own !split commands)
output in tmp_copyright.rst
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'sphinx', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|CC BY}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to sphinx
*** error: heading "===== Exercise: My problem =====" is subsection and
    comes after chapter, but will be treated as section in sphinx
found info about 1 exercises
*** warning: new !split inserted (override all existing !split)
    before every ========= heading =========
    because this strategy gives a well-functioning
    table of contents in Sphinx
    (use --sphinx_keep_splits to enforce your own !split commands)
output in tmp_copyright.rst
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'sphinx', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|CC BY-NC}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to sphinx
*** error: heading "===== Exercise: My problem =====" is subsection and
    comes after chapter, but will be treated as section in sphinx
found info about 1 exercises
*** warning: new !split inserted (override all existing !split)
    before every ========= heading =========
    because this strategy gives a well-functioning
    table of contents in Sphinx
    (use --sphinx_keep_splits to enforce your own !split commands)
output in tmp_copyright.rst
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'sphinx', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|Released under the MIT license.}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to sphinx
*** error: heading "===== Exercise: My problem =====" is subsection and
    comes after chapter, but will be treated as section in sphinx
found info about 1 exercises
*** warning: new !split inserted (override all existing !split)
    before every ========= heading =========
    because this strategy gives a well-functioning
    table of contents in Sphinx
    (use --sphinx_keep_splits to enforce your own !split commands)
output in tmp_copyright.rst
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'html', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to html
found info about 1 exercises
output in tmp_copyright1.html
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'html', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|CC BY}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to html
found info about 1 exercises
output in tmp_copyright2.html
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'html', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|CC BY-NC}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to html
found info about 1 exercises
output in tmp_copyright3.html
running mako on tmp_copyright.do.txt to make tmp_mako__tmp_copyright.do.txt
mako variables: {'FORMAT': 'html', 'DEVICE': 'screen', 'COPYRIGHT': '{copyright|Released under the MIT license.}', 'BOOK': True}
Translating doconce text in tmp_mako__tmp_copyright.do.txt to html
found info about 1 exercises
output in tmp_copyright4.html
