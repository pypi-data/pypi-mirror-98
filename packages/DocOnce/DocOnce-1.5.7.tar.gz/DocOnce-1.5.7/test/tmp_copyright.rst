.. Automatically generated Sphinx-extended reStructuredText file from DocOnce source
   (https://github.com/doconce/doconce/)

.. provide COPYRIGHT="..." and BOOK=False/True on the command line

.. Document title:

Test
%%%%

:Authors: Hans Petter Langtangen (hpl at simula.no), Kaare Dump (kd at simula.no), Joe Doe
:Date: Jan 32, 1999

.. !split

My first chapter
%%%%%%%%%%%%%%%%

.. math::
         a = b 

A section
=========

Write text.

.. !split

.. _sec:theory:

Basic Theory
%%%%%%%%%%%%

A first discovery was that 1+1 is 2.

.. --- begin exercise ---

Exercise 1: My problem
----------------------

Here is an important equation:

.. math::
   :label: my:prob:eq1

        
        2+2 = 4
        
        

As shown in :eq:`my:prob:eq1`, 2 and 2 is 4.

.. --- end exercise ---
