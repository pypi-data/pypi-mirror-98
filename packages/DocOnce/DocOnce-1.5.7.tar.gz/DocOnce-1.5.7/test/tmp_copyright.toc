\contentsline {chapter}{\numberline {1}My first chapter}{3}{chapter.1}
\contentsline {section}{\numberline {1.1}A section}{4}{section.1.1}
\contentsline {chapter}{\numberline {2}Basic Theory}{5}{chapter.2}
