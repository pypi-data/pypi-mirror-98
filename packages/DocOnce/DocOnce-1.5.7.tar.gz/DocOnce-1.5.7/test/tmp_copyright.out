\BOOKMARK [0][-]{chapter.1}{My first chapter}{}% 1
\BOOKMARK [1][-]{section.1.1}{A section}{chapter.1}% 2
\BOOKMARK [0][-]{chapter.2}{Basic Theory}{}% 3
